src/OrbitGl/TimerTrack.cpp

```diff
@@ -271,8 +271,8 @@ void TimerTrack::UpdatePrimitives(Batcher* batcher, uint64_t min_tick, uint64_t
   draw_data.ns_per_pixel = time_window_ns / viewport_->GetScreenWidth();
   draw_data.min_timegraph_tick = time_graph_->GetTickFromUs(time_graph_->GetMinTimeUs());
 
-  for (auto& chain : chains_by_depth) {
-    if (!chain) continue;
+  for (auto* chain : chains) {
```

Comment:
I know it was there before, but I'm not a huge fan of auto :-) Consider using TimerChain instead.

Reply:
done.